helpers/providers/azure.ts

```diff
@@ -26,6 +27,9 @@ const ALL_AZURE_OPENAI_CHAT_MODELS: Record<string, { openAIModel: string }> = {
   "gpt-4o-2024-05-13": {
     openAIModel: "gpt-4o-2024-05-13",
   },
+  "gpt-4o-mini-2024-07-18": {
```

Comment:
@marcusschiesser please help me to check if this model version is right! I'm just checking an online document to add this.

Reply:
https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#how-do-i-access-the-gpt-4o-and-gpt-4o-mini-models